profile/src/main/java/org/openedx/profile/presentation/manageaccount/compose/ManageAccountView.kt

```diff
@@ -177,7 +177,6 @@ internal fun ManageAccountView(
                                         borderColor = MaterialTheme.appColors.primaryButtonBorderedText,
                                         textColor = MaterialTheme.appColors.primaryButtonBorderedText
                                     )
-                                    Spacer(modifier = Modifier.height(12.dp))
                                     IconText(
```

Comment:
Do we need to implement a ripple effect or show a border? [Reference](https://www.figma.com/design/q6z6cpJsbBBcnPZLPS7KEu/Marketplace-Visual-Design?node-id=6545-485&t=Sq8pgMPGpvpixzWb-0)

Reply:
I have already made this change in https://github.com/edx/edx-mobile-marketplace-android/pull/32 for Delete button effect. that's why I skipped it in this PR